Solutions/Vectra AI Stream/Data/Solution_Vectra.json

```diff
@@ -0,0 +1,29 @@
+{
+  "Name": "Vectra AI Stream",
+  "Author": "Microsoft - support@microsoft.com",
+  "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/AIVectraDetect.svg\"width=\"75px\"height=\"75px\">",
+  "Description": "The [AI Vectra Stream](https://www.vectra.ai/) solution allows to send Network Metadata collected by Vectra Sensors across the Network and Cloud to Microsoft Sentinel.**Underlying Microsoft Technologies used:** \r \n This solution takes a dependency on the following technologies, and some of these dependencies either may be in [Preview](https://azure.microsoft.com/support/legal/preview-supplemental-terms/) state or might result in additional ingestion or operational costs:  a. [Agent based logs collection from Windows and Linux machines](https://docs.microsoft.com/azure/azure-monitor/agents/data-sources-custom-logs)",
```

Comment:
Remove hyperlink for `AI Vectra Stream` from `Description` as there is no link required as per document

![Uploading image.png…]()